### PR TITLE
Fixes a problem that prevented a redirection to the marker/pattern pane when the user didn't own assets

### PR DESCRIPTION
**Pull request by javierarce**
This PR redirects to right icon tab when the user doesn't have any asset of his/her own.

Original issue: https://github.com/CartoDB/cartodb/issues/4084

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4683/))

**Comment by javierarce** [comment]
@xavijam: I need your :eyeglasses: here, please.

File: lib/assets/javascripts/cartodb/common/dialogs/map/image_picker_view.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ module.exports = BaseDialog.extend({
     var items = this.collection.where({ kind: this.kind });
 
     if (items.length === 0) {
-      if (this.kind == 'markers') {
+      if (this.kind == 'marker') {
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/common/dialogs/map/image_picker_view.js]: ===

**Comment by xavijam** [comment]
^^^ and go!

**Comment by javierarce** [comment]
retest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4687/))

**Comment by saleiva** [comment]
bump!

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4772/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4773/))

**Comment by javierarce** [comment]
Merged with master. I'll deploy it tomorrow morning.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4818/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4828/))

**Pull request merged**